src/pages/Home.js

```diff
@@ -62,26 +62,30 @@ export default function Home(props) {
 
   return (
     <main>
-      <h1>Welcome To Honey Do</h1>
-      <p>Your Smart Shopping List.</p>
+      <h1 className="mt-5 text-3xl text-center self-start font-light">
+        Welcome to Honey Do. Your smart shopping list.
+      </h1>
       <img
         src={honeyDoCurve}
         alt="colorful circular logo with half of a honeydew melon as a shopping basket with grocery items coming out of it"
-        className="mt-5 mb-5"
+        className="p-8"
```

Comment:
Everything looks AMAZING @apdsrocha 

But I didn't love the HUGE logo on desktop. So I changed line 71 to:
```javascript
 className="md:max-w-md md:m-auto"
```
And I think that really helps the overall home page look on desktop and doesn't affect how it looks on mobile at all! 

Let me know what you think!

Reply:
I think you're amazing at Tailwind, that's what I think 😆  I struggled with the huge logo too, but had no idea how to make it better and thought it might be too late to go into that rabbit hole. Thank you so much for having such a quick fix!

Reply:
Thanks for the kind words Ana! I've been using tailwind for a couple months now so I've picked up a few tricks here and there lol.

Anyway, this looks amazing! 

SO PR APPROVED!!!!